src/components/App/App.js

```diff
@@ -22,9 +22,12 @@ function App() {
         description={Organization.DESCRIPTION}
       />
       <Facts data={RECIDIVIZ_FACTS} />
-      {/* TODO: This will be wrapped in a Recidiviz component in next PR */}
       {RecidivizList.map(recidiviz =>
-        <RecidivizIntro description={recidiviz.description} key={recidiviz.type} />)}
+        (<Recidiviz
+          type={recidiviz.type}
+          description={recidiviz.description}
+          key={recidiviz.type}
```

Comment:
Where is the `key` used?

Reply:
Best practice for React, it's used to make DOM re-rendering more efficient. Ref: https://reactjs.org/docs/lists-and-keys.html

Reply:
Read through and it makes sense now.